### PR TITLE
[it] Localizzazione termine mTLS

### LINKED ISSUE #1603: [it] Localizzazione Mutual Transport Layer Security (mTLS)
https://glossary.cncf.io/mutual-transport-layer-security/

### PR DESCRIPTION
**Pull request by ugho16**
### Describe your changes
Added mTLS Italian localization

### Related issue number or link (ex: `resolves #issue-number`)
Closes #1603 

### Checklist before opening this PR (put `x` in the checkboxes)
- [x] This PR does not contain plagiarism
  - don’t copy other people’s work unless you are quoting and contributing it to them.
- [x] I have signed off on all commits 
  - [signing off](https://docs.github.com/en/authentication/managing-commit-signature-verification/signing-commits) (ex: `git commit -s`) is to affirm that commits comply [DCO](https://wiki.linuxfoundation.org/dco). If you are working locally, you could add an alias to your `gitconfig` by running `git config --global alias.ci "commit -s"`.

### CODE REVIEW

File: content/it/mutual-transport-layer-security.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+## Quali problematiche affronta
+
+I [microservizi](/it/microservices/) comunicano tramite una rete e,
+proprio come in una rete Wi-Fi domestica, le comunicazioni in transito su quella rete possono essere violate.
+Il mTLS assicura che nessun soggetto non autorizzato possa mettersi in ascolto o impersonare richieste legittime.
+
+## In che modo aiuta
+
+Il mTLS garantisce che il traffico sia sicuro e affidabile in entrambe le direzioni tra un client e il server,fornendo un livello di sicurezza per gli utenti che accedono a una rete o a delle applicazioni.
+Inoltre, verifica che le connessioni con dispositivi client che non seguono un processo di accesso, come i dispositivi Internet of Things (IoT).
```
Comment (sistella) [inline content/it/mutual-transport-layer-security.md:23]:
```suggestion
Inoltre, verifica le connessioni con dispositivi client che non seguono un processo di accesso, come i dispositivi Internet of Things (IoT).

```

**Review by sistella — CHANGES_REQUESTED** [review]
Grande! solo un'inezia ...

**Review by sistella — APPROVED** [review]

File: content/it/mutual-transport-layer-security.md [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+
+## Cos'è
+
+Il Mutual TLS (mTLS) è una tecnica utilizzata per autenticare e criptare i messaggi inviati tra due [servizi](/it/service/).
+Il Mutual TLS è il protocollo [Transport Layer Security](/it/transport-layer-security/) (TLS) standard,
+ma invece di validare l'identità di una sola connessione entrambi i lati sono validati.
+
+## Quali problematiche affronta
+
+I [microservizi](/it/microservices/) comunicano tramite una rete e,
```
Comment (annalisag-spark) [inline content/it/mutual-transport-layer-security.md:16]: Eviterei di andare a capo dopo la virgola

**Review by annalisag-spark — CHANGES_REQUESTED** [review]

File: content/it/mutual-transport-layer-security.md (hunk 2 above)
Comment (ugho16) [reply content/it/mutual-transport-layer-security.md:17]: è soltanto un modo per tenere corte le righe in Markdown e favorire i cambiamenti singoli, lato front-end viene tenuto tutto nella stessa riga

**Review by annalisag-spark — APPROVED** [review]

**Pull request merged**